alpaca/trading/models.py

```diff
@@ -107,6 +107,19 @@ class Position(BaseModel):
     change_today: str
 
 
+class AllAccountsPositions(BaseModel):
+    """
+    Represents the positions of every account as of last market close.
+
+    Attributes:
+        as_of (datetime): Timestamp for which the positions are returned.
+        positions (dict[str, List[Position]]): Positions held by an account, keyed by account_id.
+    """
+
+    as_of: datetime
+    positions: dict[str, List[Position]]
```

Comment:
Wasn't entirely sure what the best way was to parse the response from the API, given the sample JSON:

```json
{
  "as_of": "2022-08-04T16:00:00-04:00",
  "positions": {
    "000c8a47-7487-430b-94e1-e628a71cd123": [
      {
          "asset_id": "b0b6dd9d-8b9b-48a9-ba46-b9d54906e415",
          "symbol": "AAPL",
          "exchange": "NASDAQ",
          "asset_class": "us_equity",
          "asset_marginable": true,
          "qty": "0.079145874",
          "avg_entry_price": "172.34",
          "side": "long",
          "market_value": "13.14850404762",
          "cost_basis": "13.63999992516",
          "unrealized_pl": "-0.49149587754",
          "unrealized_plpc": "-0.0360334223047464",
          "unrealized_intraday_pl": "0",
          "unrealized_intraday_plpc": "0",
          "current_price": "166.13",
          "lastday_price": "166.13",
          "change_today": "0",
          "qty_available": "0.079145874"
      },
    ]
  }
}
```

Reply:
Also what naming convention you guys would use with regards to the `AllAccountsPositions` class, if you have a better suggestions 😅

Reply:
Hey that naming is great! It's concise. 

Also it looks like there are a few missing parameters like `asset_marginable`, `qty_available`, `swap_rate`, `avg_entry_swap_rate` and `usd` that show up in the sample response but not in the `Position` class. 

We should add them in with the following types

```
asset_marginable: Optional[boolean]
qty_available: Optional[str]
swap_rate: Optional[str]
avg_entry_swap_rate: Optional[str]
```


`usd` is a `USDPosition` object that represents the position in USD values and has the following schema

```
USDPositionValues {
	avg_entry_price:  str
	market_value: str
	cost_basis: str
	unrealized_pl: str
	unrealized_plpc: str
	unrealized_intraday_pl: str
	unrealized_intraday_plplc: str
	current_price: str
	lastday_price: str
	change_today: str
}
```

We should add that as in a separate class. As for docstrings, we can just copy over the descriptions from the corresponding parameters in the `Position` class.

Reply:
Awesome, thanks for the review!

I've added `USDPositionValues` as a separate class here:
https://github.com/alpacahq/alpaca-py/pull/268/files#diff-0b073dbc392a75dd00da5d6797304d38a600da4f4465ed46bdf385805fad324bR69-R96

And subsequently added `usd` as a property to the `Position` class here:
https://github.com/alpacahq/alpaca-py/pull/268/files#diff-0b073dbc392a75dd00da5d6797304d38a600da4f4465ed46bdf385805fad324bR147

I also had to update all the mock test responses to include the `usd` parameter since it's required now:
https://github.com/alpacahq/alpaca-py/pull/268/files#diff-9632831dc1447b5ae3d486c18c6af501bbd0077bcbeeacf9c0e1fe5ed4f04080R101-R112

Reply:
Also, I managed to find existing references to document to all the parameters you have mentioned to add except for `avg_entry_swap_rate`:
https://github.com/alpacahq/alpaca-py/pull/268/files#diff-0b073dbc392a75dd00da5d6797304d38a600da4f4465ed46bdf385805fad324bR122

If you don't mind advising what this should be 😅